Dockerfile

```diff
@@ -433,6 +433,7 @@ RUN ln -s /usr/local/lib64/libCGAL.so.13 /usr/local/lib && \
         cmake \
         linux-headers && \
     cd /tmp && \
+    rm -rf build && \
```

Comment:
Is this necessary? Did you have issues there? Normally all `/tmp` is cleared after each build stage.

Reply:
Yes, but that's a different thing.
Here we copy the whole source directory, and if the source directory contains a `build` directory (often the case), then `mkdir` fails.

https://github.com/vvmruder/3dfier/blob/a18af880e4e2bfda571b7168b72c030851432187/Dockerfile#L416

Reply:
Ok i agree. But I added it in .dockerignore https://github.com/tudelft3d/3dfier/pull/88/files#diff-f7c5b4068637e2def526f9bbc7200c4eR1
So it should be omitted on the copy command from docker file. Isn't it the case?

Reply:
Hm I see. It still copied the `build` for me...
Oh wait :man_facepalming: 
I didn't copy `.dockerignore` when I switched branch :grimacing: 
I'll try again.

Reply:
Alrite, `.dockerignore` is indeed doing what it's meant to.